python/tvm/contrib/util.py

```diff
@@ -69,15 +73,15 @@ def listdir(self):
         return os.listdir(self.temp_dir)
 
 
-def tempdir():
+def tempdir(custom_path=None):
     """Create temp dir which deletes the contents when exit.
 
     Returns
```

Comment:
document the parameter custom_path and the behavior when it is specified

Reply:
Added ```custom-path``` description for the public tempdir().